test-coverage/lcov-report/base.css

```diff
@@ -0,0 +1,224 @@
+body, html {
```

Comment:
We definitely don't want to commit the `test-coverage/` folder. Look into something called the `.gitignore` file and see if you can remove these files using gitignore.

I am going to hold off review until we can get this PR cleaned up a bit because we definitely don't want to merge as is.